### ISSUE TITLE
Multiple accounts and services (i.e. Mastodon feed)

### ISSUE THREAD

**Issue by gungfujoe** [mike-ward/tweetz#134]
Enhancement request: Any chance Tweetz might be updated to provide both a user's Twitter feed and their Mastodon feed?  I love Tweetz as an unobtrusive background desktop widget, and I haven't found anything nearly as good for Mastodon.

**Comment by mike-ward** [comment]
Does Mastodon have an API? Don't know anything about it.

**Comment by gungfujoe** [comment]
Yeah, Mastodon is an open source Twitter-like app.  Their API and documentation are available at https://docs.joinmastodon.org/client/ , though a quick search suggests that there may also be a number of third-party APIs. Mastodon has some more feed views than Twitter, but a simple "home" feed (those you follow, much like the Twitter feed) would be great to put on my desktop!

**Comment by mike-ward** [comment]
I have project called, "Loon" which is a cross platform version of tweetz. At present the text rendering in Avalonia (the UI framework is not quite ready for production. If I were to extend to other accounts/services it would likely be there.

**Comment by erottensteiner** [comment]
Yes - it would quite be great if tweetz could provide a mastodon feed!! I love tweetz, it's the perfect information-bar on my right side of the desktop. If "loon" is ready for first release, I would give it a try - if it can replace using tweetz...?

**Comment by mike-ward** [comment]
I'm more inclined to support http://post.news when they publish an API. I've tried Mastodon and I'm not a fan of how it works.

**Comment by gungfujoe** [comment]
Since there are multiple services popping up to compete with Twitter, I titled the issue "Multiple accounts and services," to allow for other possibilities. I hadn't yet made it past Post's waitlist when I wrote the Issue, but now that I have, I still prefer Mastodon. It seems a lot more Twitter-like to me, and a better replacement (despite its numerous shortcomings).  Post is nice, but it's more like Blogger-lite or Wordpress-lite, with a lot of very long posts. I do use both, though, and if I had a Tweetz/Luna-like app that allowed me to see them _both_ scroll by in a sidebar window on my desktop (intermingled with my Twitter feed, if that API ever works again), I'd be ecstatic. :)

I know many are also concerned about Post's profit motive, but until I see evidence that is actually causing problems, I don't really share that concern.
